packages/ui/src/components/Catalog/CatalogFilter.tsx

```diff
@@ -0,0 +1,57 @@
+import { Form, FormGroup, Grid, GridItem, SearchInput, ToggleGroup, ToggleGroupItem } from '@patternfly/react-core';
+import { FunctionComponent, useEffect, useRef } from 'react';
+
+interface CatalogFilterProps {
+  className?: string;
+  searchTerm: string;
+  groups: string[];
+  activeGroup: string;
+  onChange: (event: unknown, value?: string) => void;
+  setActiveGroup: (group: string) => void;
+}
+
+export const CatalogFilter: FunctionComponent<CatalogFilterProps> = (props) => {
+  const inputRef = useRef<HTMLInputElement>(null);
+
+  useEffect(() => {
+    inputRef.current?.focus();
+  }, []);
+
+  return (
+    <Form className={props.className}>
+      <Grid hasGutter md={6}>
+        <GridItem>
+          <FormGroup label="Search term" fieldId="search-term">
+            <SearchInput
+              aria-label="Find by name, description or tag"
+              placeholder="Find by name, description or tag"
```

Comment:
still description mentioned here?

Reply:
I forgot to push the changes :face_in_clouds: